widgetssdk/src/main/java/com/glia/widgets/chat/ChatContract.kt

```diff
@@ -23,7 +23,7 @@ internal interface ChatContract {
         fun setView(view: View)
         fun getView(): View?
         fun onDestroy(retain: Boolean)
-        fun onMessageClicked(messageId: String)
+        fun onTapToRetryClicked(messageId: String)
```

Comment:
A bit lost if the 'Tap' and Clicked' in the same method name intentional and I just don't get it or just confusing naming.
If it is not intentional I would suggest `onMessageRetryClicked` to make it very clear that it is not just some retry but message retry

Reply:
renamed - c64be489fee6ea4819df6498f1bd8e2ab9977e99